nas_ports/freenas/freenas-migrate113/Makefile

```diff
@@ -1,7 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	freenas-migrate113
-PORTVERSION=	2019112901
+PORTVERSION=	2020000000
+PORTREVISION=	1
```

Comment:
Dont need the revision, fwiw

Reply:
It will be the one that is incremented every time